Yarn install `Packing the package failed exit code 1`

I add this line `"wa-sqlite": "github:rhashimoto/wa-sqlite#semver:^0.9.8"` to `dependencies` in `package.json`.

I'm using Yarn PnP.

My terminal:

```sh
yarn install                                                                                                                                    1 ↵
➤ YN0000: · Yarn 4.1.0
➤ YN0000: ┌ Resolution step
➤ YN0058: │ wa-sqlite@https://github.com/rhashimoto/wa-sqlite.git#commit=390744d41c61aa0bbd53d3c738abef5e23f71cc4: Packing the package failed (exit code 1, logs can be found here: /private/var/folders/ys/spt316fs5sxck01gjy40s1rc0000gn/T/xfs-116092da/pack.log)
➤ YN0000: └ Completed in 7s 917ms
➤ YN0000: · Failed with errors in 7s 924ms
```

Logs:
```
Packing wa-sqlite@https://github.com/rhashimoto/wa-sqlite.git#commit=390744d41c61aa0bbd53d3c738abef5e23f71cc4 from sources
Using Yarn for bootstrap. Reason: "__metadata" key found in yarn.lock

➤ YN0036: Calling the "prepack" lifecycle script
➤ YN0000: wa-sqlite@workspace:. STDERR make: *** No rule to make target `tmp/obj/dist/sqlite3.o', needed by `dist/wa-sqlite.mjs'.  Stop.
➤ YN0036: Prepack script failed (exit code 2, logs can be found here: /private/var/folders/ys/spt316fs5sxck01gjy40s1rc0000gn/T/xfs-76bcc0d5/prepack.log); run yarn prepack to investigate
➤ YN0000: Failed with errors in 0s 10ms
```

--- a/README.md
+++ b/README.md
@@ -27,7 +27,7 @@ Here are the build steps:
 * `git clone git@github.com:rhashimoto/wa-sqlite.git`
 * `cd wa-sqlite`
 * `yarn install`
-* `make` (or `yarn prepack`)
+* `make`
 
 The default build produces ES6 modules + WASM, [synchronous and asynchronous](https://github.com/rhashimoto/wa-sqlite/issues/7) (using Asyncify) in `dist/`.
 

--- a/package.json
+++ b/package.json
@@ -15,7 +15,6 @@
   ],
   "scripts": {
     "build-docs": "typedoc",
-    "prepack": "make",
     "start": "web-dev-server --node-resolve",
     "test": "karma start karma.conf.cjs --browsers ChromeHeadless",
     "test-manual": "karma start karma.conf.cjs --auto-watch --no-single-run --reporters progress"

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "wa-sqlite",
-  "version": "0.9.11",
+  "version": "0.9.12",
   "type": "module",
   "main": "src/sqlite-api.js",
   "types": "src/types/index.d.ts",

